zfs/replicate/cli/main.py

```diff
@@ -52,13 +52,13 @@
 @click.option(
     "--cipher",
     type=EnumChoice(Cipher),
-    default=Cipher.STANDARD,
+    default=Cipher.STANDARD.name.lower(),
     help="One of: disable (no ciphers), fast (only fast ciphers), or standard (default ciphers).",
 )
 @click.option(
     "--compression",
     type=EnumChoice(Compression),
-    default=Compression.LZ4,
+    default=Compression.LZ4.name.lower(),
```

Comment:
I could have sworn that the default should be of the same type of the final variable itself.

Wouldn't it be easier to check in `EnumChoice.convert()` if the value is already of the same type to shortcircuit? That way this part of the code is not changed.

Reply:
I removed the commit that adds this.  I'll merge this and we can follow up on specifics in other pull reqeusts.